### PR TITLE
[NRF] Added port for uBlox Nina B1 series BLE modules

### PR DESCRIPTION
**Pull request by rolandvs**
uBlox has a series of `NINA` modules with the same footprint. One of them is the NINA-B1 with a Nordic nRF52832. They also feature ESP32 based modules.

### CODE REVIEW

**Comment by dpgeorge** [comment]
Thanks, it looks ok.  I assume you tested that the REPL works, and things like LEDs?  (I don't have such a board so can't test it myself.)

**Comment by rolandvs** [comment]
Yes, leds, repl, uart, w/ w/o s132, ble. Will do some more testing. Even though there is rgb led support, there are no constants to create colours...

**Comment by dpgeorge** [comment]
> Yes, leds, repl, uart, w/ w/o s132, ble.

Ok great!  So that's good enough testing to start with.

Merged in ac112f88d0496d33a9195ace02ed3928fcb015d3

File: ports/nrf/boards/evk_nina_b1/mpconfigboard.h [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+#define MICROPY_PY_RANDOM_HW_RNG    (1)
+
+#define MICROPY_HW_HAS_LED          (1)
+#define MICROPY_HW_LED_TRICOLOR     (1)
+#define MICROPY_HW_LED_PULLUP       (1)
+
+#define MICROPY_HW_LED_RED          (8)   // LED1 DS8 Red
+#define MICROPY_HW_LED_GREEN        (16)  // LED2 DS8 Green
+#define MICROPY_HW_LED_BLUE         (18)  // LED3 DS8 Blue
+// LEDs conflict with UART
```
[the remark this answers is not in the document]
Comment (glennrub) [reply ports/nrf/boards/evk_nina_b1/mpconfigboard.h]: Ah, understand. Thanks for explaining this to me =)

**Pull request closed without merging**